#+STARTUP: indent align fold hideblocks hidestars
#+OPTIONS: reveal_center:nil reveal_control:t reveal_height:-1
#+OPTIONS: reveal_history:nil reveal_keyboard:t reveal_overview:t
#+OPTIONS: reveal_progress:t reveal_rolling_links:nil
#+OPTIONS: reveal_single_file:nil reveal_slide_number:"c"
#+OPTIONS: num:nil 
#+OPTIONS: toc:nil
#+OPTIONS: reveal_title_slide:nil reveal_width:-1
#+REVEAL_MARGIN: -1
#+REVEAL_MIN_SCALE: -1
#+REVEAL_MAX_SCALE: -1
#+REVEAL_ROOT: ./
#+REVEAL_TRANS: fade
#+REVEAL_SPEED: default
#+REVEAL_THEME: simple
# #+REVEAL_EXTRA_CSS: css/reveal.css
#+REVEAL_EXTRA_CSS: css/custom-qike.css
#+REVEAL_EXTRA_CSS: css/tomorrow.css
# #+REVEAL_EXTRA_CSS: plugin/toc-progress/toc-progress.css
#+REVEAL_EXTRA_JS: 
#+REVEAL_HLEVEL:1
#+REVEAL_TITLE_SLIDE_BACKGROUND:
#+REVEAL_TITLE_SLIDE_BACKGROUND_SIZE:
#+REVEAL_TITLE_SLIDE_BACKGROUND_POSITION:
#+REVEAL_TITLE_SLIDE_BACKGROUND_REPEAT:
#+REVEAL_TITLE_SLIDE_BACKGROUND_TRANSITION:
#+REVEAL_DEFAULT_SLIDE_BACKGROUND:
#+REVEAL_DEFAULT_SLIDE_BACKGROUND_SIZE:
#+REVEAL_DEFAULT_SLIDE_BACKGROUND_POSITION:
#+REVEAL_DEFAULT_SLIDE_BACKGROUND_REPEAT:
#+REVEAL_DEFAULT_SLIDE_BACKGROUND_TRANSITION:
#+REVEAL_MATHJAX_URL: https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML
#+REVEAL_PREAMBLE:
#+REVEAL_HEAD_PREAMBLE:
#+REVEAL_POSTAMBLE:
#+REVEAL_MULTIPLEX_ID:
#+REVEAL_MULTIPLEX_SECRET:
#+REVEAL_MULTIPLEX_URL:
#+REVEAL_MULTIPLEX_SOCKETIO_URL:
#+REVEAL_SLIDE_HEADER:
#+REVEAL_SLIDE_FOOTER:
#+REVEAL_PLUGINS: (classList markdown highlight zoom notes multiplex)
#+REVEAL_DEFAULT_FRAG_STYLE:
# #+REVEAL_INIT_SCRIPT: { src: 'plugin/toc-progress/toc-progress.js', async: true, callback: function() { toc_progress.initialize(); toc_progress.create(); } }
#+REVEAL_HIGHLIGHT_CSS: %r/lib/css/zenburn.css
# #+REVEAL_INIT_SCRIPT:.slide: style="text-align: left;"

#+EMAIL:     qikeli@gmail.com
#+TITLE: WNAR-2017-Qike-Li
#+AUTHOR:    Qike Li 

# the following set up for external plugins.
# 
#+BEGIN_SRC emacs-lisp :exports none :eval no
;; To eable the third party plug-in:toc-progress, I need to remove "no" 
;: after :eval for this code block and C-c C-c to evaluate this code block. 
;; In addtion, line 19, containing 'toc-progress.css', needs to be un-commented
;; To turn this feature off, run elisp :(custom-set-variables '(org-reveal-external-plugins nil))
(custom-set-variables '(org-reveal-external-plugins '((toc-progress . "{src: 'plugin/toc-progress/toc-progress.js', async: true, callback: function() { toc_progress.initialize(); toc_progress.create(); }}") ))) 
#+END_SRC


** iDEG: a new single-subject method for assessing gene differential expression from two transcriptomes of an individual

    #+BEGIN_EXPORT html
    <hr class="shadow">
    #+END_EXPORT
Qike Li 

qike.li@gmail.com

[[https://qikeli.github.io][qikeli.github.io]]

#+ATTR_HTML: :align left :height 110px
[[./Figures/statistics-gidp.png]]

#+ATTR_HTML: :align right :height 110px
[[./Figures/ua_logo.png]]

#+ATTR_HTML: :align center :height 110px
[[./Figures/Lussier.png]]

* Outline
@@html:<br>@@
- Background  
- Methodology--iDEG
- Numerical studies
- Real data analysis
* Background
- Precision Medicine
- Single-subject analysis
- Example RNA-Seq quantified mRNA expression data (Single-Subject)
- Our goal
- Main Challenges
- Our solution (iDEG)

** Precision Medicine 
#+ATTR_HTML: :align center 
   [[./Figures/PGX.jpg]]  
 
** Single-subject analysis
@@html:<br><br>@@
 #+ATTR_HTML: :style background-color:lightgray
  @@html:<span class="fragment highlight-blue">@@Single-subject analysis@@html:</span>@@ studies  @@html:<span class="fragment highlight-blue">@@one patient@@html:</span>@@ at a time, with the purpose of revealing the  @@html:<span class="fragment highlight-blue">@@patient-specific@@html:</span>@@ disease mechanism for highly heterogeneous diseases.
** Single-subject analysis
 #+ATTR_HTML: :align center 
 [[./Figures/single-subject-analysis2.png]]
** Example RNA-Seq quantified mRNA expression data (Single-Subject)
@@html:<br>@@
#+BEGIN_SRC R :exports none :results output :session :eval no
load('~/Dropbox/Qike/adaptive_cutoff/Figures/Fig_BRCA_TNBC/Data/TCGA-GI-A2C9.RDat')

exp_tnbc_A2C9 <- round(exp_tnbc_A2C9)
table_tnbc_A2C9 <- data.frame(Gene = rownames(exp_tnbc_A2C9),
                              Case_expression = exp_tnbc_A2C9$Tumor_Sample,
                              Baseline_expression = exp_tnbc_A2C9$Healthy_Sample)

table_tnbc_A2C9_print <- head(table_tnbc_A2C9,10)
table_tnbc_A2C9_print <- ascii::ascii(table_tnbc_A2C9_print,
               header = T,
               include.rownames = F,
               include.colnames = T,
               format = 'd',
               frame = 'topbot',
               caption = 'TNBC example')

print(table_tnbc_A2C9_print, type = 'org')
#+END_SRC

#+CAPTION: Single-subject RNA-Seq data example
|--------+-----------------+---------------------|
| Gene   | Case expression | Baseline expression |
|--------+-----------------+---------------------|
| /      |               < |                     |
| A1BG   |              92 |                  72 |
| A1CF   |               1 |                   0 |
| A2BP1  |               0 |                   2 |
| A2LD1  |             127 |                  71 |
| A2ML1  |             773 |                  12 |
| A2M    |           11825 |               29385 |
| A4GALT |             871 |                 891 |
| A4GNT  |               1 |                   5 |
| AAA1   |               0 |                   0 |
| AAAS   |             414 |                 460 |
| ...    |             ... |                 ... |
|--------+-----------------+---------------------|


#+BEGIN_EXPORT html
<div class="reference"> 
<br><br><br><br>
<font><hr></hr><sup> Table 1: Data source <a href="https://portal.gdc.cancer.gov/">https://portal.gdc.cancer.gov/</a></sup> </font>
</div>
#+END_EXPORT
#+REVEAL: split
#+ATTR_HTML: :height 600px
[[./Figures/giphy.gif]]
** Our goal 
#+ATTR_REVEAL: :frag (appear)
 - How differentially expressed is a gene?
   - A meaningful effect size of differential expression
   
 - What is the uncertainty of the measurement?
   - A probability of gene differential expression.
@@html:<br>@@
** Main Challenges
@@html:<br><br>@@
@@html:<div style="background-color:lightgray;">@@
#+ATTR_HTML: :align left
Main challenges in single-subject analysis:
 - There are @@html:<font color="red">@@NO@@html:</font>@@  within subject replicates.
# @@html:</div>@@

#+REVEAL: split


#+ATTR_HTML: :align center :height 600px
[[./Figures/nightmare_cartoon.png]]

** Main Challenges

@@html:<div style="background-color:lightgray;">@@
#+ATTR_HTML: :align left
Main challenges in single-subject analysis:
- There are @@html:<font color="red">@@NO@@html:</font>@@  within subject replicates.
  - Different genes have different variances. 
  - For each gene, we have only two numbers to work with.

#+BEGIN_SRC R :exports results :results graphics :file ./Figures/fig-tnbc.png :session :eval no
base_size <- 20
dat_plot <- table_tnbc_A2C9[,2:3]
     ## plot data

## p_pois_fc <- 
##     ggplot(data = dat_plot, 
##            aes(x = rowMeans(dat_plot), y = log2(Case_expression + 1) - log2(Baseline_expression+1))) +
##     geom_point(alpha = trans_value) +
##     xlim(0,10000)+
##     labs(x = TeX('$\\mu_{g1}$'), y= TeX('$log_2(Y_{g2} + 1) - log_2(Y_{g1} + 1)$')) 
## p_pois_fc 

p_tnbc <- 
    ggplot(data = dat_plot, 
           aes(x = rowMeans(dat_plot), y = Case_expression - Baseline_expression)) +
    geom_point(alpha = .6) +
    xlim(0,4000)+
    ylim(-5000,5000)+
    labs(x = latex2exp::TeX('$\\frac{Case + Baseline}{2}$'), y= 'Case - Baseline') +
    theme_bw(base_size = base_size)
p_tnbc
#+END_SRC

** Our solution (iDEG)
@@html:<br>@@
#+ATTR_REVEAL: :frag (appear)
- *Bypass gene-level variance estimation*: Transform RNA-Seq data such that, under null hypothesis, all genes have the same variance.
- *Borrow strength across genes*: model all genes in a single model
* Methodology--iDEG
@@html:<br>@@
- Data transformation
- Two-group mixture model

** Data transformation
#+BEGIN_SRC R :exports none :results output :session :eval no
rm(list=ls())
 
## load in ggplot objects
load('~/Dropbox/Qike/adaptive_cutoff/Figures/Fig_examples/Results/fig-examples.RData', verbose = T)


base_size <- 20
## panel 1
p1 <- p_pois +
    theme_bw(base_size = base_size) +
    guides(col = guide_legend(title = "DEG status", title.position = "left"),
           shape = guide_legend(title = "DEG status", title.position = "left")) +
    ggtitle('Before Transformation') + 
    theme(plot.margin = unit(c(2.5,.5,0.5,1.2), "lines"),
          plot.title = element_text(hjust = 0.5,lineheight=.8, face="bold")) 



## panel 2
p2 <- p_pois_tran +
    theme_bw(base_size = base_size) +
    guides(col = guide_legend(title = "DEG status", title.position = "left"),
           shape = guide_legend(title = "DEG status", title.position = "left")) +
    ggtitle('After Transformation') +
    theme(plot.margin = unit(c(2.5,.5,0.5,1.2), "lines"),
          plot.title = element_text(hjust = 0.5,lineheight=.8, face="bold")) +
    labs(y = latex2exp::TeX('$h(Y_{g2}) - h(Y_{g1})$'))


p_comb <- cowplot::plot_grid(p1 + theme(legend.position = 'none'),
                             p2 + theme(legend.position = 'none'),
                             ncol = 2)
#+END_SRC

#+RESULTS:
: Loading objects:
:   p_pois
:   p_pois_tran
:   p_pois_iDEG
:   p_nb
:   p_nb_tran
:   p_nb_iDEG
:   p_pois_fdr

#+BEGIN_SRC R :exports none :results graphics :file ./Figures/fig-beforeVST.png :width 500 :height 600 :session :eval no
plot(p1 + theme(legend.position = 'bottom') + ggtitle(''))
#+END_SRC

#+RESULTS:
[[file:./Figures/fig-beforeVST.png]]

[[./Figures/fig-beforeVST.png]]

** Data transformation

#+BEGIN_SRC R :exports none :results graphics :file ./Figures/fig-trans.png :session :width 850 :height 600 :eval no
cowplot::plot_grid(p_comb, get_legend(p1 + theme(legend.position = 'bottom')),ncol = 1, rel_heights = c(3, .3)) 
#+END_SRC

#+RESULTS:
[[file:./Figures/fig-trans.png]]

[[./Figures/fig-trans.png]]

** Data transformation
@@html:<br><br>@@

#+ATTR_HTML: :align left
Under null hypothesis:

@@html:<br>@@

#+ATTR_HTML: :style font-size:90% 
 \begin{eqnarray}
 \qquad Z_{g} &=& h(Y_{g2}) - h(Y_{g1}) \overset{\cdot}{\sim} \text{N}(0,c) \\
    && \text{  where } c \text{ is an unknown constant}
\end{eqnarray}

** Two-group mixture model
#+ATTR_HTML: :align left
Suppose $G$ genes are measured in the RNA-Seq experiment, each of the genes is either null or differentially expressed with prior probabilities $\pi_{0}$ or $\pi_{1} = 1-\pi_{0}$. And the density function of $z_{g}$ is either $f_{0}(z)$ or $f_{1}(z)$.

#+ATTR_HTML: :style font-size:90% 
\begin{eqnarray*}
      \pi_{0} &=& Pr\{\mbox{gene $g$ is null}\} \\
    \pi_{1} &=& Pr\{\mbox{gene $g$ is DEG}\}
\end{eqnarray*}

#+ATTR_HTML: :align left
The marginal mixture density is:
#+ATTR_HTML: :style font-size:90% 
\begin{equation*}
  \label{eq:mixtureDens}
  f(z) = \pi_{0}f_{0}(z) + \pi_{1}f_{1}(z)
\end{equation*}

** Two-group mixture model
#+ATTR_HTML: :align left
The local false positive rate, $fdr$, is the Bayes posterior probability that a gene $g$ is a null gene given $z_{g}$: 
@@html:<br><br>@@
#+ATTR_HTML: :style font-size:90% 
\begin{equation*}
  \label{eq:locfdr}
  fdr(z) \equiv Pr\{\text{null gene} | z\} = \pi_{0}f_{0}(z)/f(z)
\end{equation*}

** Two-group mixture model
#+ATTR_HTML: :align left :style font-size:90%
$f_{0}$ corresponds to an empirical null distribution. 
#+ATTR_HTML: :align left :style font-size:90%
In large-scale simultaneous hypothesis testing, the theoretical null may be deficient due to various reasons: 

- correlation across genes

- unobserved covariates (e.g. gender, age, smoking status, etc.)

- or failed mathematical assumptions (e.g. asymptotic approximation).
#+ATTR_HTML: :align left :style font-size:90%
 Fortunately, in large-scale simultaneous testing, the parallel structure allows the estimation of an empirical null distribution, via empirical Bayes, from the own data of the study. 


* Numerical studies
- Simulate a single-subject dataset
- Procedure
- Method evaluation 
- Sensitivity Analysis

** Simulate a single-subject dataset
#+ATTR_HTML: :align left
Simulate a pair of transcriptomes:
#+ATTR_HTML:  :style font-size:90%
\begin{eqnarray*}
  Y_{g1} &\sim& \text{NB}(\mu_{g1}, \delta_{g})\\
  Y_{g2} &\sim& \text{NB}(\mu_{g2}, \delta_{g})\\ 
  \delta_{g} &=& 0.005 + 9/(\mu_{g1}+100) \\
P(\mu_{g1}) &=& \frac{1}{500} e^{-\frac{1}{500} \mu_{g1}} \quad \quad g = 1,\cdots,20\,000
 \end{eqnarray*}


** Simulate a single-subject dataset

#+BEGIN_EXPORT html
<div align="right"> 
<font style="background-color:lightgray"> <i> Cont'd </i> </font>
</div>
#+END_EXPORT

#+ATTR_HTML: :style font-size:90% 
\begin{eqnarray*}
\mu_{g2} &=& \begin{cases}
\mu_{g1}  & \text{if }g \in \overline{\mathcal{G}},\\
d^{s}\mu_{g1}   & \text{if }g \in \mathcal{G}.\end{cases}\\
&&\text{where $\mathcal{G}$ is a random subset of set $\{1,2, \cdots, 20000\}$}\\
\frac{|\mathcal{G}|}{|\mathcal{G}| + |\overline{\mathcal{G}}|} &=& 0.05 \label{eq:DEG-pct}\\
d  &=& \frac{\mu_{g1} + n \sqrt{\mu_{g1}}}{\mu_{g1}}\label{eq:DEG-fold-change}\\
s &=& \begin{cases} 1 &\text{with probability of }0.5,\\
-1 &\text{with probability of }0.5.\end{cases} \label{eq:fold-change-direction}\\
n &\sim& \mathcal{N}(9,1)
\end{eqnarray*}

** Procedure
- **Step 1:** Simulate one single-subject dataset, which contains $p \%$ DEG. 
- **Step 2:** Conduct iDEG, edgeR[fn:1:], DESeq[fn:2:], and DEGseq[fn:3:].
- **Step 3:** Compute an F1 score for each method,
#+ATTR_HTML: :style font-size:90%
\begin{equation*}
F_{1} = \frac{2\times Precision \times Recall}{Precision + Recall}   
\end{equation*}
- **Step 4:** Repeat Step1-Step3 for 1000 times
- **Step 5:** Calculate the arithmetic mean and standard deviation of the 1000 $F_{1}$ resulted from each method.
- **Step 6:** Change the value of $p$, repeat Step 1-Step 5

#+BEGIN_EXPORT html
<div class="reference"> 
<font ><hr></hr><sup>1. Robinson, M. D. et. al., 2010, <i>Bioinformatics</i>;  2. Anders, S. et. al., 2010, <i> Genome biology </i>;<br /> 3. Wang, L. et al., 2009, <i>Bioinformatics</i></sup> </font>
</div>
#+END_EXPORT


** Method evaluation 

#+BEGIN_SRC R :exports none :results graphics :file ./Figures/fig-F1.png :session width :res 120 :height 550 :width 600
load('~/Dropbox/Qike/adaptive_cutoff/Figures/Fig_numericalStudy/Results/fig-num-study2.RData', verbose = T)
p_num_study + theme_bw(base_size = 15) + 
theme(legend.position = 'top',
 strip.background = element_blank(),
       strip.text = element_blank())
#+END_SRC

#+RESULTS:
[[file:./Figures/fig-F1.png]]


[[./Figures/fig-F1.png]]

** Method evaluation 

#+ATTR_HTML: :table rules="groups"
| DEG%  | Method | Precision                                        | Recall(TPR)                                       | FPR             | F1              | Predictions        |
|-------+--------+--------------------------------------------------+---------------------------------------------------+-----------------+-----------------+--------------------|
|    5% | iDEG   | @@html:<font color="red">0.93</font>@@ (1.6e-02) | @@html:<font color="red">0.679</font>@@ (2.5e-02) | 0.003 (7.2e-04) | 0.784 (1.5e-02) | 730.482 (3.4e+01)  |
|-------+--------+--------------------------------------------------+---------------------------------------------------+-----------------+-----------------+--------------------|
|       | edgeR  | @@html:<font color="red">0.39</font>@@ (8.4e-03) | @@html:<font color="red">0.948</font>@@ (7.1e-03) | 0.078 (2.8e-03) | 0.552 (8.6e-03) | 2432.95 (5.4e+01)  |
|-------+--------+--------------------------------------------------+---------------------------------------------------+-----------------+-----------------+--------------------|
|       | DESeq  | 0.999 (2.1e-03)                                  | 0.152 (3.8e-02)                                   | 0 (1.8e-05)     | 0.262 (5.8e-02) | 152.45 (3.8e+01)   |
|-------+--------+--------------------------------------------------+---------------------------------------------------+-----------------+-----------------+--------------------|
|       | DEGseq | 0.086 (6.7e-04)                                  | 0.985 (3.9e-03)                                   | 0.549 (3.9e-03) | 0.159 (1.2e-03) | 11408.61 (7.4e+01) |
|   10% | iDEG   | 0.945 (1.1e-02)                                  | 0.708 (2.2e-02)                                   | 0.005 (1.1e-03) | 0.809 (1.2e-02) | 1500.32 (5.9e+01)  |
|-------+--------+--------------------------------------------------+---------------------------------------------------+-----------------+-----------------+--------------------|
|       | edgeR  | 0.447 (6.2e-03)                                  | 0.96 (4.3e-03)                                    | 0.132 (3.3e-03) | 0.61 (6.0e-03)  | 4296.19 (6.0e+01)  |
|-------+--------+--------------------------------------------------+---------------------------------------------------+-----------------+-----------------+--------------------|
|       | DESeq  | 1 (0.0e+00)                                      | 0 (5.2e-04)                                       | 0 (0.0e+00)     | 0.002 (1.4e-03) | 0.44 (1.0e+00)     |
|-------+--------+--------------------------------------------------+---------------------------------------------------+-----------------+-----------------+--------------------|
|       | DEGseq | 0.165 (1.1e-03)                                  | 0.986 (2.5e-03)                                   | 0.556 (4.2e-03) | 0.282 (1.6e-03) | 11974.99 (7.6e+01) |
|   15% | iDEG   | 0.953 (7.0e-03)                                  | 0.746 (1.6e-02)                                   | 0.006 (1.1e-03) | 0.837 (9.1e-03) | 2348.99 (5.8e+01)  |
|-------+--------+--------------------------------------------------+---------------------------------------------------+-----------------+-----------------+--------------------|
|       | edgeR  | 0.537 (5.7e-03)                                  | 0.964 (3.7e-03)                                   | 0.147 (3.4e-03) | 0.69 (4.8e-03)  | 5384.34 (5.9e+01)  |
|-------+--------+--------------------------------------------------+---------------------------------------------------+-----------------+-----------------+--------------------|
|       | DESeq  | 1 (NA)                                           | 0 (3.3e-05)                                       | 0 (0.0e+00)     | 0.001 (NA)      | 0.01 (1.0e-01)     |
|-------+--------+--------------------------------------------------+---------------------------------------------------+-----------------+-----------------+--------------------|
|       | DEGseq | 0.235 (1.4e-03)                                  | 0.986 (2.1e-03)                                   | 0.565 (4.2e-03) | 0.38 (1.9e-03)  | 12562.45 (7.3e+01) |
|   20% | iDEG   | 0.962 (4.6e-03)                                  | 0.763 (1.3e-02)                                   | 0.008 (1.0e-03) | 0.851 (7.8e-03) | 3175 (6.4e+01)     |
|-------+--------+--------------------------------------------------+---------------------------------------------------+-----------------+-----------------+--------------------|
|       | edgeR  | 0.602 (5.7e-03)                                  | 0.966 (2.8e-03)                                   | 0.16 (3.9e-03)  | 0.742 (4.4e-03) | 6418.81 (6.4e+01)  |
|-------+--------+--------------------------------------------------+---------------------------------------------------+-----------------+-----------------+--------------------|
|       | DESeq  | NaN (NA)                                         | 0 (0.0e+00)                                       | 0 (0.0e+00)     | NaN (NA)        | 0 (0.0e+00)        |
|-------+--------+--------------------------------------------------+---------------------------------------------------+-----------------+-----------------+--------------------|
|       | DEGseq | 0.299 (1.6e-03)                                  | 0.986 (2.0e-03)                                   | 0.577 (4.2e-03) | 0.459 (1.9e-03) | 13179.58 (6.8e+01) |

#+REVEAL: split

#+BEGIN_EXPORT html
<div align="right"> 
<font style="background-color:lightgray"> <i> Cont'd </i> </font>
</div>
#+END_EXPORT
@@html:<br>@@
#+ATTR_html: :style font-size:80% :align left
Although the Recall/TPR and number of precisions of iDEG are lower than edgeR, iDEG has high precision and low FPR across all percentages of DEG. These operating characteristics of iDEG may be preferable in large-scale inference, like RNA-Seq analysis, where investigators examine tens of thousands of genes in a high-throughput manner.

** Sensitivity Analysis

#+ATTR_html: :align left
Our experience indicates that without making assumptions on RNA-Seq data, it is difficult to construct suitable statistical inferences for a single-subject dataset. Nonetheless we test the two main assumptions we make:

- The value of dispersion parameter is a function of expression mean.

- The majority of the genes are null genes. 

** Sensitivity Analysis
  - Draw  dispersion parameter $\delta_{g}$ from a uniform distribution $\text{unif}(0.001, 0.1)$. 
[[./Figures/fig-sensitivity-random.png]]

** Sensitivity Analysis
 - Simulate single-subject datasets with a series of percentages of DEG. 

 [[./Figures/fig-sensitivity-highDEG.png]]

* Real data analysis
- Data
- iDEG result
- Some interesting genes

** Data 
#+ATTR_HTML: :align left
A transcriptome of tumor sample and a transcriptome of surrounding healthy sample of an African American woman with Triple negative breast cancer (TNBC). 

#+ATTR_HTML: :align left :style font-size:85%
TNBC is a subtype of breast cancer that has poor prognosis and considerable heterogeneity. It disproportionately affects women with African origin (Dietze et al., 2015). Personalized treatment may be successful for an African American TNBC patient if one could discover a set of her specific DEG and match some of these DEG to the targets of a therapeutic drug.
** iDEG result
#+CAPTION: The 10 top-hit (smallest local fdr) differentially expressed genes.
#+NAME: table:tnbc
| Gene    | local_fdr |      Z |
|---------+-----------+--------|
| ADIPOQ  |  2.85e-34 | -11.17 |
| PLA2G2A |  2.85e-34 | -11.65 |
| PI16    |  1.15e-33 | -10.78 |
| LEP     |  2.25e-33 | -10.70 |
| SFTPB   |  1.44e-32 | -10.59 |
| IL33    |  4.24e-31 | -10.36 |
| TUSC5   |  6.74e-31 | -10.32 |
| CSF3    |  2.89e-29 | -10.04 |
| COL6A6  |  3.24e-29 | -10.04 |
| CCL21   |  1.99e-28 |  -9.90 |
| ...     |       ... | ...    |
** Some interesting genes

#+ATTR_HTML: :style font-size:65%
- *ADIPOQ* & *LEP*: Adiponectin (gene product of gene ADIPOQ) and leptin (gene product of LEP) are considered as mediators for the association of breast cancer with obesity, which is a major risk factor for breast cancer (Nalabolu et al., 2014; Grossmann et al., 2010).
- *PLA2G2A*: Although PLA2G2A has not been extensively studied for breast cancer, many studies have shown that it inhibits invasion and metastasis of gastric and colon cancer (Ganesan et al., 2008; Movahedian et al., 2016; Fijneman et al., 2009) and may predict survival (Xing et al., 2011).
- *TUSC5*: Bubnov et al. (2012) has demonstrated the down-regulation of TUSC5 induced by DNA methylation in breast cancer. In contrast to mutated genes, DNA methylation is reversible.
* Take home message
@@html:<br>@@
#+ATTR_HTML: :align left :style background-color:lightgray;font-size:120% 
We developed a new and effective method to identify differentially expressed genes from two transcriptomes of a single patient.    

* Acknowledgements

#+BEGIN_EXPORT html
<div class="sliderow">

<div class="leftcol">
<br>
<img src="http://www.stat.ncsu.edu/people/photos/hao_zhang.jpg"  style="float:left;width:35%;height:45%;"/> 
<p > Helen Hao Zhang </p>
<br>
<br>
<br>
<br>
<img src="http://deptmedicine.arizona.edu/sites/default/files/styles/1x_smartphone/public/lussier.jpg?itok=n-R0y_3l&timestamp=1481818852" style="float:left;width:35%;height:45%;" align="left"/>
<p > Yves A. Lussier </p>
</div>


<div class="rightcol">

<br>
<ul>
<li> Haiquan Li</li>
<li> Joanne Berghout</li>
<li> Grant Schissler</li>
<li> Yuan Shang</li>
</ul>
<br>
<br>
<img src="https://www.nsf.gov/images/logos/nsf1.jpg"  height="130" width="130"/> <font color="white">spa</font>
<img src="https://www.nih.gov/sites/default/files/styles/sidebar_media_breakpoint-small/public/about-nih/2012-logo.png?itok=WGmazXcU"  height="130" width="130"/>
<img src="http://grad.arizona.edu/ui/images/ua_lockups/gidp-lockup-full-white.png" height="100" width="400" style="background-color:gray" />
</div>

</div>
#+END_EXPORT
